scipy/stats/_axis_nan_policy.py

```diff
@@ -579,13 +607,15 @@ def hypotest_fun_out(*samples, **kwds):
                 return tuple_to_result(*res)
 
             # Addresses nan_policy == "omit"
+            too_small_msg = too_small_nd_omit
```

Comment:
I think this assignment is redundant? We only have one user in the rest of the function, which could just use `too_small_nd_omit` directly